scout/server/blueprints/cases/controllers.py

```diff
@@ -1,13 +1,16 @@
 # -*- coding: utf-8 -*-
 import itertools
+import datetime
 
 from flask import url_for
 from flask_mail import Message
 import query_phenomizer
 
 from scout.constants import (CASE_STATUSES, PHENOTYPE_GROUPS, COHORT_TAGS)
+from scout.constants.variant_tags import MANUAL_RANK_OPTIONS, DISMISS_VARIANT_OPTIONS, GENETIC_MODELS
 from scout.models.event import VERBS_MAP
 from scout.server.utils import institute_and_case
+from scout.server.blueprints.variants.controllers import variants_description, variants_filter_by_field
 
 STATUS_MAP = {'solved': 'bg-success', 'archived': 'bg-warning'}
 SEX_MAP = {'1': 'male', '2': 'female'}
```

Comment:
There is a SEX_MAP in [case_tags.py](https://github.com/Clinical-Genomics/scout/blob/master/scout/constants/case_tags.py).
please use that one 😺